db/migrate/20151435234623_add_container_command.rb

```diff
@@ -0,0 +1,5 @@
+class AddContainerCommand < ActiveRecord::Migration
+  def change
+    add_column :container_definitions, :command, :string
```

Comment:
The default column width for type `:string` is `256`, I believe.  For the [command line](https://github.com/ManageIQ/manageiq/blob/88efa888d9528cdbf853894ceee455c416800585/db/migrate/20130923182042_collapsed_initial_migration.rb#L1787) field in `miq_workers`, it uses `:limit => 512`.

I don't know how long the `command` value is supposed to be, but be aware of the column limits in place here.